pkgs/applications/networking/cluster/cert-manager-ctl/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+
+buildGoModule rec {
+  pname = "cert-manager-ctl";
+  version = "1.6.1";
```

Comment:
I think we can init it with the latest version (1.7.0) https://github.com/jetstack/cert-manager/releases/tag/v1.7.0

Reply:
Done